akka-javasdk-tests/src/test/java/akkajavasdk/JwtEndpointTest.java

```diff
@@ -4,65 +4,159 @@
+    
+    CompletableFuture<StrictResponse<String>> call = httpClient.GET("/hello").addHeader("Authorization", token)
+        .responseBodyAs(String.class)
+        .invokeAsync().toCompletableFuture();
 
-        Awaitility.await()
-                .ignoreExceptions()
-                .atMost(5, TimeUnit.SECONDS)
-                .untilAsserted(() -> {
-                    assertThat(call.get().body()).isEqualTo("issuer: my-issuer-123, subject: my-subject-123");
-                });
+    Awaitility.await()
+        .ignoreExceptions()
+        .atMost(5, TimeUnit.SECONDS)
+        .untilAsserted(() -> {
+          assertThat(call.get().body()).isEqualTo("issuer: my-issuer-123, subject: my-subject-123");
+        });
 
-    }
+  }
+
+  @Test
+  public void shouldReturnForbidden() {
+    var token = bearerTokenWith(Map.of("iss", "my-issuer-123"));
+
+    var call = httpClient.GET("/hello").addHeader("Authorization", token)
+        .invokeAsync().toCompletableFuture();
+
+    Awaitility.await()
+        .ignoreExceptions()
+        .atMost(5, TimeUnit.SECONDS)
+        .untilAsserted(() -> {
+          assertThat(call.get().status()).isEqualTo(StatusCodes.FORBIDDEN);
+        });
+  }
+
+  @Test
+  public void shouldRaiseExceptionWhenAccessingJWT() {
+    var token = bearerTokenWith(Map.of("iss", "my-issuer-123"));
+
+    var call = httpClient.GET("/missingjwt").addHeader("Authorization", token)
+        .responseBodyAs(String.class).invokeAsync().toCompletableFuture();
+
+    Awaitility.await()
+        .atMost(5, TimeUnit.SECONDS)
+        .untilAsserted(() -> {
+          var exception = assertThrows(Exception.class, call::get);
+          assertThat(exception.getCause().getClass()).isEqualTo(RuntimeException.class);
+          assertTrue(exception.getCause().getMessage().contains(
+              "There are no JWT claims defined but trying accessing the JWT claims. The class or the method needs to be annotated with @JWT."));
+        });
+
+  }
 
-    @Test
-    public void shouldReturnForbidden() {
-        var call = httpClient.GET("/hello").addHeader("Authorization","Bearer eyJhbGciOiJIUzI1NiIsImtpZCI6ImtleTEifQ.eyJpc3MiOiJteS1pc3N1ZXItMTIzIn0.7d3PJvLIFA22nVW9eNIMJhkR9m5oInTwrRuwtaNNzB8")
-                .invokeAsync().toCompletableFuture();
-
-        Awaitility.await()
-                .ignoreExceptions()
-                .atMost(5, TimeUnit.SECONDS)
-                .untilAsserted(() -> {
-                    assertThat(call.get().status()).isEqualTo(StatusCodes.FORBIDDEN);
-                });
-    }
 
-    @Test
-    public void shouldRaiseExceptionWhenAccessingJWT() {
-        var call = httpClient.GET("/missingjwt").addHeader("Authorization","Bearer eyJhbGciOiJIUzI1NiIsImtpZCI6ImtleTEifQ.eyJpc3MiOiJteS1pc3N1ZXItMTIzIn0.7d3PJvLIFA22nVW9eNIMJhkR9m5oInTwrRuwtaNNzB8")
-                .responseBodyAs(String.class).invokeAsync().toCompletableFuture();
+  // from here down, JWT validation tests for gRPC endpoints
```

Comment:
The changes **above** this line are only about formatting (from 4 spaces to 2) and replacing the hardcoded token with a generated one, for better readability and consistency.

Reply:
Huh, we don't have javafmt set up here? If not we should have, but unrelated to this PR